.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+
+python:
+ - 2.7
+ - 3.5
+
+env:
+  - TOXENV=django18
+  - TOXENV=django19
+  - TOXENV=django110
+
+sudo: false
+
+branches:
+  only:
+  - master
+
+install:
+  - pip install -r requirements/test_requirements.txt
+
+script:
+  - tox
+
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Huh, hadn't seen the bash uploader before, but it looks like it's now the [recommended setup](https://github.com/codecov/example-python#whats-the-different-between-the-codecov-bash-and-codecov-python-uploader)